src/engine/enginebuffer.cpp

```diff
@@ -227,6 +227,18 @@ EngineBuffer::EngineBuffer(const QString& group,
     m_pCueControl = new CueControl(group, pConfig);
     addControl(m_pCueControl);
 
+    for (int i = 1; i <= kMacrosPerChannel; ++i) {
+        auto* pMacroControl = new MacroControl(group, pConfig, i);
```

Comment:
Looks like this might currently be leaked. Perhaps should be replaced by a unique_ptr

Reply:
I remove the macro specific QList. Now it handled as all the other controls in EngineBuffer and released by qDeleteAll(m_engineControls); in the deconstructor.

Reply:
It's deleted by qDeleteAll(m_engineControls); in the deconstructor, as all other enginebuffer controls.